### DISCUSSION TITLE
各記事の下の方にある「関連記事」欄のアイコン

### DISCUSSION THREAD

**Discussion by tsubolabo** [increments/qiita-discussions#429] [要望・アイデア・不具合]

**Comment by tanaka-asami415** [comment]
@tsubolabo 
ご報告ありがとうございます🙏
「関連記事」は外部のレコメンドエンジンを利用しているため、問合せをいたします🙇

**Reply by tanaka-asami415** [reply]
@tsubolabo 
ご報告が遅くなりました🙇
アイコンが変わってしまった件については、原因が判明したため現在対応中です。
完了しましたら、改めてお知らせさせていただきます！

アイコンのクリック後の遷移については、以前からもジャンプしない仕様でした。

> また、最近、「関連記事」とは言いながら、関連性が全く分からない記事ばかりが表示されることもあります。（良く分かりませんが、未ログイン状態のときに多い？）
さらに、ときには、灰色の記事リストっぽいレイアウトが表示されるだけで、いくら待っても画面が確定しないことも多いようです。

こちらに関しては、具体的な事例がないと確認が難しいようです🙏

**Reply by kskwtnk** [reply]
@tsubolabo 
レコメンドエンジンの提供元に連絡した後、直接の解消報告はいただけておらず共有が遅くなりました。申し訳ありません
tsubolaboさんにご確認いただいたように現在は解消しているようです

灰色のリストの件は我々の方で再現しておりませんが、報告だけはしていますので一旦解消を待つということでご了承いただけますと幸いです

---

話は変わりますが、以下のコメントについて

> 貴コメントに thumbs up している tanaty5828 さんは tanaka-asami415 さん御自身なんですか？

まず回答としては「違います。名前が少し似ている別人です。」となります。

回答以外として、以下を申し上げます。

一般的に「複数アカウントを使って賛同者が多いように見せる」行為は誉められたものではありません。
今回、tanaty5828さんにはただGitHubの機能を使っているだけで「Qiita運営の自演の加担者」のように名前を挙げられてしまっています。

我々運営は、ユーザーの皆様からの意見を広く集めたいため、多少のことであれば受け入れる気持ちを持っています。
ですが、他の参加者の方に対して自演の容疑をかけるなどの行為はおやめください。

**Reply by kskwtnk** [reply]
@tsubolabo 

> 提案に異論があれば、文章で教えて欲しいものと不満に思っていた

> 私の対応も待たず、訳の分からない thumbs up マークを付けてきています。

> このように紛らわしいマークだけでの異論の表現は止めて、文章で直接的に意思表示して頂きたいものです。

このように思われるのはご自由ですが、インターネット全般、あるいはGitHub上においてはアイコンだけでリアクションをするのはありふれたコミュニケーションです。
これはQiita運営だからこのように申し上げるわけではなく、（少なくともGitHubを媒介にしている限り）遍く普及している話です。

tsubolaboさんご自身の信条として「アイコンだけの反応はせず、文章を添える」を貫かれるのでしたら廉直で素晴らしいですが、他の方に強いるのはおやめくださいませ。

**Reply by yoshinoyuya** [reply]
@tsubolabo さん
こちらの件ですが、似たような状況として、AdBlock拡張機能が影響していた事例を知っているのですが
拡張機能がない状態で試してみると、解決するかもしれません。
![image](https://user-images.githubusercontent.com/85545443/221064279-8cb2bde3-b376-465f-82ac-8379d7e11c57.png)

拡張機能なしの状態で試す方法としてはEgdeでは「InPrivateブラウザ」や「ゲストプロファイル」などがあるかと思います。


AdBlockの件の詳細

1. Qiita該当箇所は、関連記事以外に広告が表示される。
1. AdBlockが広告の読み込みを阻害。
1. 読み込みが正常に終わらず、灰色の状態から変わらない。
